tests/functional/tests/conftest.py

```diff
@@ -1,10 +1,38 @@
 # coding: utf-8
 """Configurations for py.test runner"""
-
+import pytest
+from ctrlibrary.threatresponse import token
 from ctrlibrary.core import settings
+from threatresponse import ThreatResponse
 
 
 def pytest_collection_modifyitems():
     if not settings.configured:
         settings.configure()
     return settings
+
+
+@pytest.fixture(scope='module')
+def module_token():
+    return token.request_token(
+        settings.server.ctr_client_id, settings.server.ctr_client_password)
+
+
+@pytest.fixture(scope='module')
+def module_headers(module_token):
+    return {'Authorization': 'Bearer {}'.format(module_token)}
```

Comment:
There is cleaner way to interpolate strings using f-strings:
```suggestion
    return {'Authorization': f'Bearer {module_token}'
```

Reply:
Hi, it was done to make code compatible with different versions on python, we provide support  up to python 2.7 for this particular library(you can see  it in travis file).